### DISCUSSION TITLE
How to select one texture component such as cube with deviation of 15 degrees and plot it on IPF maps

### DISCUSSION THREAD

**Discussion by Brian-Liu-rx** [mtex-toolbox/mtex#1761] [Ask Anything]
Hi ,everyone,
How to select one texture component such as cube with deviation of 15 degrees and plot it on IPF maps
In other word, filter the IPF maps by orientations

**Comment by lfgmorales** [comment]
Hi

Can you explain what do you mean by “cube”?

You may find something here

<https://mtex-toolbox.github.io/ODFComponents.html>
ODF Component Analysis | MTEX<https://mtex-toolbox.github.io/ODFComponents.html>
mtex-toolbox.github.io<https://mtex-toolbox.github.io/ODFComponents.html>
        [favicon.png] <https://mtex-toolbox.github.io/ODFComponents.html>



Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 10 Mar 2023, at 04:50, Brian-Liu-rx ***@***.***> wrote:

﻿

Hi ,everyone,
How to select one texture compnents such as cube with deviation of 15 degrees and plot it on IPF maps?
In other word, filter the IPF maps by orientations

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1761>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN7F6BR74SVTTLAEB2LW3KQHNANCNFSM6AAAAAAVV5I4WU>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1761", "url": "https://github.com/mtex-toolbox/mtex/discussions/1761", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by Brian-Liu-rx** [reply]
Hi, I mean  ploting the subset of specific texture component on the IPF map, such as cube，S or copper.
:)

**Comment by maguag** [comment]
Hi, maby you can try this code in my former discussion：

https://github.com/mtex-toolbox/mtex/discussions/1654#discussioncomment-4144577
